### DISCUSSION TITLE
Should pcmanfm-qt not work outside of lxqt?

### DISCUSSION THREAD

**Discussion by Rabcor** [lxqt/lxqt#2571] [General]
The pcmanfm-qt readme says:
> PCManFM-Qt is used by LXQt for handling the desktop. Nevertheless, it can also be used independently of LXQt and with any desktop environment, whether under X11 or under Wayland.

Is that just outdated information? Or am i just not understanding it right?

I made a bug report https://github.com/lxqt/pcmanfm-qt/issues/1903

But it was closed on the grounds of "doesn't happen on lxqt" or perhaps more specifically "doesn't happen if you have lxqt installed and set environment variables to make it look like you are using lxqt and not whatever DE you're actually on", then i was told to bring it up in discussions. And well, here's the discussion i suppose...

**Comment by tsujan** [comment]
Already answered at the linked page.
